packages/application-shell/src/components/navbar/navbar.js

```diff
@@ -130,28 +134,48 @@ MenuExpander.propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
-export const MenuGroup = props => (
-  <ul
-    className={classnames(
-      { [styles.list]: props.level === 1 },
-      { [styles.sublist]: props.level === 2 },
-      {
-        [styles['sublist-no-children']]: props.level === 2 && !props.children,
-      },
-      {
-        [styles['sublist-expanded__active']]:
-          props.level === 2 && props.isActive && props.isExpanded,
-      },
-      props.level === 2 && props.isActive && !props.isExpanded
-        ? styles['sublist-collapsed__active']
-        : styles.sublist__inactive
-    )}
-  >
-    {props.children}
-  </ul>
-);
+export const MenuGroup = props => {
+  const isSublistActiveWhileIsMenuExpanded =
+    props.level === 2 && props.isActive && props.isExpanded;
+  const isSublistActiveWhileIsMenuCollapsed =
+    props.level === 2 && props.isActive && !props.isExpanded;
```

Comment:
would it make sense to move these numeric values into a const e.g `SUB_MENU_LEVEL`

Reply:
Yeah, but this navbar component needs to be rewritten to be honest. I don't want to start refactoring too much unless we actively work on it.